docs/proposals/20200804-windows-support.md

```diff
@@ -0,0 +1,271 @@
+For Beta we should be able to identify common overlapping features that can be added directly to the kubeadm cluster provisioning 
+(CABPK) in a similar way that the [kubeadm-bootstrap-script.sh](https://github.com/kubernetes-sigs/cluster-api/blob/master/bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script.sh) is done today.
+
+#### netbios names
+Netbios name resolution concerns were addressed in https://github.com/kubernetes-sigs/cluster-api/issues/2217#issuecomment-598999436
+
+### Infrastructure provider implementation
+By leveraging cloudbase-init, an infrastructure provider implementation will require only a few changes which include:
+
+- Make changes to their provider api to enable Windows OS worker nodes 
+- Ensuring the cloudbase-init data lands in the correct place on the machines for Windows OS worker nodes
+
+From the infrastructure provider perspective, there are no known required changes to the CAPI API to support Windows
+nodes at this time.  If during alpha we identify changes we will open issues pertaining to the changes required.
+
+### User Stories
+
+#### As an operator, I would like to create Windows OS worker nodes with the CAPI API.
+
+#### As an operator, I would like to manage Windows OS worker nodes with the CAPI API.
+
+### Implementation Details/Notes/Constraints
+
+Due to the lack of privileged containers in Windows there are two options for configuring the components such as 
+kube-proxy, kubelet.  The above solution using wins is preferred because it makes the move to privileged containers 
+straightforward as a drop and replace. 
+
+While this is the best choice for the alpha and the community direction there are some infrastructure providers that may 
+not be able to use wins due to signing or security concerns since wins allows the execution of any arbitrary command on 
+the host. Pre/post commands can be used as an alternative with additional scripts cached on the VHD that enable the 
+configuration.
+
+#### Signing of the components.  
+Some infrastructure providers will require any scripts and binaries are signed before deployment.  
+This will be managed by providing the ability to provide external scripts and binaries during the VHD building process.  We will also be discussing this further with the Image Builder repository team during the process for adding the VHDs.
+
+#### Known prototypes and prior work: 
+- https://github.com/adelina-t/cloudbase-init-capz-demo
+- https://github.com/benmoss/kubeadm-windows/tree/master/cluster-api-aws
+
+### Security Model
+
+Wins is currently [recommended way to use kubeadm](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/).  
+Limiting access to the named pipes that are required is one way to mitigate access.   Wins is currently 
+required during and after provisioning for running kube-proxy and the CNI daemonset.  
+
+The security model for Privileged containers is still being discussed in KEP and is still early and will be
```

Comment:
Maybe explicitly state security concerns are not in scope for alpha but will be for beta?

Reply:
I've updated the wording PLTA